presto-main/src/main/java/com/facebook/presto/resourcemanager/ResourceManagerClusterStateProvider.java

```diff
@@ -236,6 +243,17 @@ else if (!info.getState().isDone()) {
         }, e -> e.getValue().getNodeStatus().getMemoryInfo()));
     }
 
+    private boolean isConsistent()
```

Comment:
Shall we call this `validateCoordinatorConsistency`? We can directly throw inside this function.